File: src/class-problems/palindrome-number/9.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function (x) {
  const strX = x.toString();
  let result = "";

  for (let i = strX.length - 1; i >= 0; i--) {
    result += strX[i];
  }

  return strX === result;
};

File: src/tier1/palindrome-number/9.py
#9. Palindrome Number

class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """

        reverse_x = ""
        str_x = str(x)

        for i in range(len(str_x) - 1, -1, -1):
            reverse_x += str_x[i]

        return reverse_x == str_x